mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -338,8 +338,10 @@ function render_global_header( $attributes = array() ) {
 
 	// Preload the menu font.
 	if ( is_callable( 'global_fonts_preload' ) ) {
-		if ( ! is_rosetta_site() ) {
-			global_fonts_preload( 'Inter', 'latin' );
+		/* translators: Subsets can be any of cyrillic, cyrillic-ext, greek, greek-ext, vietnamese, latin, latin-ext.  */
+		$subsets = explode( ',', _x( 'Latin', 'Global menu font subsets, comma separated', 'wporg' ) );
+		foreach ( $subsets as $subset ) {
```

Comment:
`foreach` is used for the case that translators update to more than one subset, for example:
`global_fonts_preload( 'Inter', 'Cyrillic, Cyrillic-ext' );`
This would fail to load the `Cyrillic-ext`. See [here](https://github.com/WordPress/wporg-mu-plugins/pull/298/files#diff-3e55f21b4a7e4e1adfa13b4471290f4cce80b2f75fe9664fa35213977f7fb49cR98)